allennlp_hub/pretrained.py

```diff
@@ -23,7 +28,7 @@ def predictor(self) -> Predictor:
 
 # TODO(Mark): Figure out a way to make PretrainedModel generic on Predictor, so we can remove these type ignores.
 
-# Models in the demo
+# Core AllenNLP models
```

Comment:
Eventually this will go down to zero.  It's probably better to not cast judgment on what is "core" and just call these "Models in the main repo".